src/test/java/org/jenkinsci/plugins/workflow/steps/scm/SCMStepTest.java

```diff
@@ -176,6 +175,17 @@ public void scmVars() throws Exception {
         });
     }
 
+    @Test public void scmParsesFakeChangeLogSCMChangelogFile() {
+        rr.then(r -> {
+            WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+            p.setDefinition(new CpsFlowDefinition(
+                    "node() {\n" +
+                            "  checkout([$class: 'FakeChangeLogSCM'])\n" +
```

Comment:
You might need to use the full class name to be able to use the version from `jenkins-test-harness` via databinding (`$class`, etc.): `org.jvnet.hudson.test.FakeChangeLogSCM`.

Reply:
How about:
```java
    @Test public void scmParsesChangelogFileFromFakeChangeLogSCM() {
        rr.then(r -> {
            WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
            p.setDefinition(new CpsFlowDefinition(
                    "node() {\n" +
                            "  checkout(new org.jvnet.hudson.test.FakeChangeLogSCM())\n" +
                            "}", false));  // had to change from true to false because not permitted
            r.buildAndAssertSuccess(p);
        });
    }
```

Reply:
Yeah that should work, if you are using direct instantiation then you I think you could also `import org.jvnet.hudson.test.FakeChangeLogSCM` at the top of the Pipeline so you can call `new FakeChangelogSCM` if that is any clearer to you.